nucypher/characters/control/specifications/fields/label.py

```diff
@@ -0,0 +1,16 @@
+from marshmallow import fields
+
+class LabelField(fields.Field):
+
+    def _serialize(self, value, attr, obj, **kwargs):
+        return value.decode('utf-8')
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        if isinstance(value, bytes):
+            return value
+        return value.encode()
+
+    def _validate(self, value):
+        return True
```

Comment:
Is some validation logic supposed to happen here?

Reply:
> Is some validation logic supposed to happen here?

Do we want to accept empty labels (`""`)? Are there other encodings that are outside UTF8?